tests/bsim/bluetooth/host/adv/long_ad/test_scripts/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+set -eu
+
+source ${ZEPHYR_BASE}/tests/bsim/sh_common.source
+
+test_name="$(guess_test_long_name)"
+
+simulation_id=${test_name}
+
+verbosity_level=2
+
+SIM_LEN_US=$((20 * 1000 * 1000))
```

Comment:
do you really need 20 whole seconds?

Reply:
I suggest we rely on early exit from the test instead. That way, the max sim length is not so important. @theob-pro can you make sure the devices exit when the test passes?